src/models/page.model.ts

```diff
@@ -118,14 +119,24 @@ export async function createOrUpdatePage(
         "xlog",
         ...(input.applications?.filter((app) => app !== "xlog") || []),
       ],
-      ...(input.slug && {
-        attributes: [
-          {
-            trait_type: "xlog_slug",
-            value: input.slug,
-          },
-        ],
-      }),
+      attributes: [
+        ...(input.slug
+          ? [
+              {
+                trait_type: "xlog_slug",
+                value: input.slug,
+              },
+            ]
+          : []),
+        ...(input.disableAISummary
+          ? [
+              {
+                trait_type: "xlog_option_disableAISummary",
```

Comment:
Using both camel case and underscore looks strange. I suggest changing it to `xlog_disable_ai_summary`.